pom.xml

```diff
@@ -32,7 +34,6 @@
     <module>dispatcher</module>
     <module>msgpack-core</module>
     <module>msgpack-value</module>
-    <module>json-path</module>
```

Comment:
The dependency to the `json-path` module is still declared in the `bom/pom.xml` as artifact `zeebe-msgpack-json-path` ;)

Reply:
It would have been too easy otherwise ;-)